server/portal/apps/workspace/api/views.py

```diff
@@ -124,9 +116,9 @@ def get(self, request, *args, **kwargs):
                 try:
                     tapis.files.listFiles(systemId=system_id, path="/")
                 except (InternalServerError, UnauthorizedError):
-                    success = _test_listing_with_existing_keypair(system_def, request.user)
+                    success = _test_listing_after_creating_credentials(system_def, request.user)
+                    logger.info(f"Unable to create credentials for {request.user} in system: {system_id}")
```

Comment:
Wouldn't this `logger.info` be dependent on the result of `_test_listing_after_creating_credentials`